crypto/ts/ts_verify_ctx.c

```diff
@@ -60,8 +60,16 @@ X509_STORE *TS_VERIFY_CTX_set_store(TS_VERIFY_CTX *ctx, X509_STORE *s)
     return ctx->store;
 }
 
+#if !OPENSSL_API_3
 STACK_OF(X509) *TS_VERIFY_CTS_set_certs(TS_VERIFY_CTX *ctx,
                                         STACK_OF(X509) *certs)
+{
+    return TS_VERIFY_CTX_set_certs(ctx, certs);
+}
+#endif
```

Comment:
@Iniesta8 @InfoHunter thinking once more about this, I think that adding a second function is a rather heavyweight way to fix the typo problem, because you end up with two different exported symbols in libcrypto.num.

If I recall correctly (@mattcaswell? @levitte?), we only promised that 3.0 is source compatible, i.e. applications don't need to be modified, but they very likely need to be recompiled.  In this case, I would recommend to do the following (in version-historical order):

1.1.1: 

Introduce the correct function as a macro:

- Add a `#define TS_VERIFY_CTX_set_certs(ctx, cert) TS_VERIFY_CTS_set_certs(ctx,cert)` from correct to the wrong name, without deprecation marker.
- Add TS_VERIFY_CTX_set_certs.pod (correct filename) with both names in the NAME section

3.0

Swap the roles of function and macro:  

- correct the function name (reusing the ordinal in libcrypto.num) and  `#define TS_VERIFY_CTS_set_certs(ctx, cert) TS_VERIFY_CTX_set_certs(ctx,cert)` from the wrong to the correct name, surrounded by deprecation marker.
- Add deprecation note to pod file

Reply:
_(The documenation for 1.1.1 could already contain a note about the type and that the correct name is preferred.)_

Reply:
s/type/typo/g

Reply:
Or maybe even better, use a macro without arguments: `#define TS_VERIFY_CTS_set_certs TS_VERIFY_CTX_set_certs`, just for the unlikely case that someone is using the address operator on the wrong function name: `&TS_VERIFY_CTS_set_certs`.

Reply:
I think the point is do we really need to fix this typo (in both code & doc) in 1.1.1?

Reply:
Or to be more accurate, is it worth doing so?

Reply:
It might not be worth doing it. For 3.0 nevertheless, I would proceed as above, i.e. add a compatibility _macro_, not a _function_.

Reply:
It seems my last email didn't reach Github, so send it again (with minor modification...):

Then there is no divergency now ;-)